tests/conftest.py

```diff
@@ -183,3 +183,14 @@ def lossy_strategy(gov, vault, create_lossy_strategy):
 def fee_manager(project, gov):
     fee_manager = gov.deploy(project.FeeManager)
     yield fee_manager
+
+
+@pytest.fixture
+def deposit_into_vault(project):
+    def deposit_into_vault(vault, account, amount=10**18):
```

Comment:
this kind of actions are expected to be implemented in the actions.py file in utils

Reply:
that would work for me. 

what's the advantage of using fixtures vs actions.py? 

the main disadvantage I see is that we need to "request" explicitely the function in the test definition. But if I am missing something I am ok to use fixtures. We just need a common approach. 

@anticlimactic what do you think?